chacractl/util.py

```diff
@@ -20,6 +21,7 @@ def load_config():
     config = get_config_path()
     if not config:
         return
+    sys.modules.pop('chacractl', None)
```

Comment:
You have a few options here (there is nothing wrong on using the fixture explicitly where you need it though): 

* mark a whole class with the fixture http://pytest.org/latest/fixture.html#using-fixtures-from-classes-modules-or-projects
* automatically use the fixture for every single test in the scope (module, function, session): http://pytest.org/latest/fixture.html#autouse-fixtures-xunit-setup-on-steroids

Does every test need it? Or just a subset? 
I understand the global scope concern here, but it has global scope because for the CLI use it is OK.